src/kv-queue/module.ts

```diff
@@ -0,0 +1,54 @@
+import { OnAppBootstrap, OnAppClose } from '../hook/interfaces.ts';
+import { MetadataHelper } from '../metadata/helper.ts';
+import { injector, Logger, Module } from '../mod.ts';
+import { KV_QUEUE_NAME, queueListenerMetadataKey } from './constants.ts';
+import { KvQueue } from './kv.ts';
+
+@Module({})
+export class KvQueueModule implements OnAppBootstrap {
+	private logger: Logger = new Logger('QueueModule');
+
+	constructor() {}
+
+	public static forRoot(kvName?: string) {
+		return {
+			injectables: [{ token: KV_QUEUE_NAME, useValue: kvName }, KvQueue],
```

Comment:
As the `kvName` is an optional value, we should consider adding a default value for it

Reply:
I do not know if we want to, given that it's also optional for `Deno.openKv` https://docs.deno.com/deploy/kv/manual